### PR TITLE
Fixes scanner gate and server shell

### LINKED ISSUE #10140: [Bug]: Scanner Gate Shell not working
### BYOND Version

514.1589

### Round Date

N/A

### Round ID

N/A

### Test merges

N/A

### Describe the bug

When I use the Scanner Gate Shell, I notice that the components do not behave as expected.

1. The Scanner Gate trigger signal does not trigger the next component
2. The Scanner Gate triggers the next component (was only able to get the next component to trigger once), but the entity of the "Scanned Object" is the floor - not the object passing through the scanner gate.
3. Other input components (such as the Voice Activator) do NOT trigger the next component when used in the Scanner Gate Shell.

### What did you expect?

I expected the Scanner Gate Shell to work as other shells and execute the circuit with the correct values.

### Reproducible steps

Scenario 1:
Screenshot 2023-11-05 at 12 59 23
1. Spawn a `Scanner Gate Shell`, `Integrated Circuit`, `Power Cell`, `To String` Component (or `Get Name`, `Get Species`), and a `Speech` Component.
2. Add power cell, and connect the circuit `Scanner Gate` -> `To String` -> `Speech`
3. Go through the gate and observe that the value of the trigger output on the `Scanner Gate` has been updated, but the `To String` component has not been triggered.

Scenario 2:
Can't replicate it anymore as the `Scanner Gate` won't trigger the next component.

Scenario 3:
image
1. Spawn a `Scanner Gate Shell`, `Integrated Circuit`, `Power Cell`, `Voice Activator` component, `To String` Component (or `Get Name`, `Get Species`), and a `Speech` Component.
2. Add power cell, and connect the circuit `Voice Activator` -> `To String` -> `Speech`. Leave the output of the `Scanner Gate` void.
3. Speak some words and observe that the `Voice Activator` does not trigger the next component. It DOES however trigger itself.

### PR DESCRIPTION
**Pull request by Dejaku51**
## About The Pull Request
Fixes #10140 
Previously you needed to secure the shell before inserting the circuit, this fixes that.
Also fixes scanner gate outputting turf it is on instead of the scanned object.
Added description to both shells saying that they need to be secured to work

## Why It's Good For The Game
Working content good

## Testing Photographs and Procedure
1. Make a circuit inside scanner gate shell that will output scanned entity
2. it will output scanned entity instead of the turf it is on

And

1. Insert any circuit into scanner gate/server shell
2. secure it
3. it works

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
tweak: Added information about needing to be secured to server and scanner gate shell
fix: Fixed scanner gate shell outputting turf instead of scanned object
fix: made securing a shell after inserting circuit work
/:cl:

### CODE REVIEW

File: code/modules/wiremod/shell/scanner_gate.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/structure/scanner_gate_shell
 	name = "circuit scanner gate"
-	desc = "A gate able to perform mid-depth scans on any organisms who pass under it."
+	desc = "A gate able to perform mid-depth scans on any organisms who pass under it. Requires to be secured to work."
```
Comment (JixS4v) [inline code/modules/wiremod/shell/scanner_gate.dm]:
```suggestion
	desc = "A gate able to perform mid-depth scans on any organisms who pass under it. Needs to be secured to work."
```
Sounds more human this way

File: code/modules/wiremod/shell/server.dm [hunk 2]
```diff
@@ -7,6 +7,7 @@
 /obj/structure/server
 	name = "server"
 	icon = 'icons/obj/wiremod.dmi'
+	desc = "A server shell able to host advanced circuits. Requires to be secured to work."
```
Comment (JixS4v) [inline code/modules/wiremod/shell/server.dm]:
```suggestion
	desc = "A server shell able to host advanced circuits. Needs to be secured to work."
```

**Review by JixS4v — CHANGES_REQUESTED** [review]

File: code/modules/wiremod/shell/scanner_gate.dm (hunk 1 above)
Comment (Dejaku51) [reply code/modules/wiremod/shell/scanner_gate.dm]: Good thing you're here because i ain't no human

**Review by JixS4v — APPROVED** [review]
Looks good

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**